org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/refactoring/RefactoringIssueAcceptor.xtend

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2017 TypeFox GmbH (http://www.typefox.io) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse def License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.ide.refactoring
+
+import org.apache.log4j.Logger
+import org.eclipse.emf.common.util.URI
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.emf.ecore.resource.ResourceSet
+import org.eclipse.xtext.util.ITextRegion
+
+/**
+ * @author koehnlein - Initial contribution and API
+ * @since 2.13
+ */
+interface RefactoringIssueAcceptor {
+	
+	enum Severity {
+		FATAL, ERROR, WARNING, INFO, OK
```

Comment:
It's a switch-case where the order should not matter
https://github.com/eclipse/xtext-eclipse/commit/241c1f73014df6af5d4dae84db745a9d6c91acab#diff-52aef468dd0def3666d5565c8b9f4df7